apps/studio/components/interfaces/Storage/AnalyticsBuckets/AnalyticsBucketDetails/NamespaceWithTables/index.tsx

```diff
@@ -282,6 +217,53 @@ export const NamespaceWithTables = ({
         visible={importForeignSchemaShown}
         onClose={() => setImportForeignSchemaShown(false)}
       />
+      {(pollIntervalNamespaceTables > 0 || missingTables.length > 0) && (
+        <CardFooter className="border-t px-4 py-4 flex flex-row justify-end gap-x-4 relative overflow-hidden">
```

Comment:
This `CardFooter` now houses all the (table and namespace) state management that was happening via the old `CardHeader`.

Reply:
No other changes.